### DISCUSSION TITLE
CPU core issues

### DISCUSSION THREAD

**Discussion by PythonCancer** [explosion/spaCy#12921] [Help: Other Questions] [answered]
Does the zh_core_web_sm-3.5.0 model of Spacy run on multi-core or single-core when performing various tasks on a CPU machine? Is it controllable? Are there any interface parameters?

**Comment by rmitsch** [comment, marked as answer]
Hi @PythonCancer, all spaCy models run multi-core when using [`.pipe()`](https://spacy.io/api/language#pipe) with `n_process != 1`.

**Reply by PythonCancer** [reply]
Based on our previous conversation, I understand that spaCy models run multi-core when using the .pipe() method with n_process != 1. However, in my case, I'm not using the .pipe() method and am calling the model directly on individual sentences. Does this mean that the processing is done using a single core by default? Could you please clarify?

**Reply by rmitsch** [reply]
Yes, that is correct.
